### ISSUE TITLE
Setting config file from S3 doesn't work 

### ISSUE THREAD

**Issue by DanCorvesor** [AdRoll/batchiepatchie#92]
I am trying to deploy batchiepatchie on ECS with Terraform. In order to have all the benefits of Terraform, I want to be able to save the Terraform code to our BitBucket but obviously that means that I don't want to save the config.toml file as it contains the RDS password. 

As a result, I wish to use the ability you have to point to the config file in S3 to get the toml file. I have never used go before but I think that you are pointing all the AWS client to us-east-1, am I correct? Would you be able to make this variable in some way as we are using eu-west-1? There are a number of places where I think you invoke the AWS Client but I'm not sure of the correct place to edit this. 

On a separate note, can you add to the documentation how you point the front end to a custom backend, I think you should use API_BASE_URL but can you confirm this and also explain what BASE_URL is. Also, it would be good to understnad the BATCHIEPATCHIE_IP in the backend, the command you use to get the local ip doesn't work. 

Finally, I am getting an error in the backend "SELECT job_queue FROM activated_job_queues : failed with error: pq: relation \"activated_job_queues\" does not exist", can you help me to debug this. I have launched the postgres db which seems to be fine and also launched the migrations as a separate task which exits with the 0 code so I'm not sure what might be wrong here.

**Comment by JonathanAquino-NextRoll** [comment]
> Finally, I am getting an error in the backend "SELECT job_queue FROM activated_job_queues : failed with error: pq: relation "activated_job_queues" does not exist", can you help me to debug this. I have launched the postgres db which seems to be fine and also launched the migrations as a separate task which exits with the 0 code so I'm not sure what might be wrong here.

Are you somehow able to see what tables are in your postgres database? activated_job_queues should have been created in migrations/00012_add_activated_job_queues.sql

**Comment by DanCorvesor** [comment]
Yep, I have done this now, I was deploying the migrations as a separate ECS task but it seems it wasn't executing properly. I would still however, like to fix this issue around the S3 config file. Are you able to help me on this please?

**Comment by JonathanAquino-NextRoll** [comment]
> As a result, I wish to use the ability you have to point to the config file in S3 to get the toml file. I have never used go before but I think that you are pointing all the AWS client to us-east-1, am I correct? Would you be able to make this variable in some way as we are using eu-west-1? There are a number of places where I think you invoke the AWS Client but I'm not sure of the correct place to edit this.

 I'm unfortunately not too familiar with this part of the code. Could you try forking the repo and changing us-east-1 to eu-west-1 to see if it helps? There seem to be 3 places.
